server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -106,6 +108,7 @@ export default class ProbationPractitionerReferralsController {
         sentReferral.referral.serviceUser.crn,
         sentReferral.referral.relevantSentenceId
       ),
+      this.assessRisksAndNeedsService.getSupplementaryRiskInformation(sentReferral.supplementaryRiskId),
```

Comment:
*Question*: the Referral details page currently fails to load if the ARN api returns a 404 for the requested Risk ID. This should only happen if there's no Risk found in the ARN api, but it should be set in the Refer journey. Is it worth adding a `catch` statement and rendering "risk not found" in this scenario?

Reply:
it would be valuable I think, yes -- it may fail with other reasons too (network death, 503, actual bug that returns 404 on everything) and it'd be nice to display something on the screen next to the field about this

Reply:
I'm wondering if it's worth doing this as a separate PR just so we at least have this in for Monday next week? Will pick it up very soon but don't want to block this one!

Reply:
We have a few places where we need to handle errors a bit more gracefully, so it could be done as part of that work.